src/main/java/com/google/sps/data/CareerQuestionDatabase.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.data;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import java.util.ArrayList;
+
+/* purpose: to manage the interface of the Career Question database */
+public final class CareerQuestionDatabase {
+ 
+  private DatastoreService datastore;
+  private ArrayList<CareerQuestionAndChoices> questionAndChoices;
+  
+  public CareerQuestionDatabase(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  /* method to return all questions and choices from database as an arraylist */
+  public ArrayList<CareerQuestionAndChoices> getQuestionsAndChoices() {
+  // TODO: replace hard coded example Q and Reponse with entities queried from datastore
+    CareerQuestionChoice HARD_CODED_CHOICE1 = new CareerQuestionChoice("choice1", "career1");
+    CareerQuestionChoice HARD_CODED_CHOICE2 = new CareerQuestionChoice("choice2", "career2");
+    ArrayList<CareerQuestionChoice> HARD_CODED_CHOICES = new ArrayList();
+    HARD_CODED_CHOICES.add(HARD_CODED_CHOICE1);
+    HARD_CODED_CHOICES.add(HARD_CODED_CHOICE2);
```

Comment:
Change local variable casing from `HARD_CODED_CHOICE1` to `hard_coded_choice_1`

Also, this should be inlined
```
    ArrayList<CareerQuestionChoice> hard_coded_choices = Arrays.asList(
        new CareerQuestionChoice("choice1", "career1"),
        new CareerQuestionChoice("choice2", "career2")
    );
    
    return Arrays.asList(
        new CareerQuestionAndChoices("A question?", hard_coded_choices));
```

Additionally, you should even set `HARD_CODED_CHOICES` as a `private static final` variable